documentation/translations/translating.rst

```diff
@@ -41,9 +41,8 @@ For more details about translations and their progress, see `the dashboard
      - Sanyam Khurana (:github-user:`CuriousLearner`)
      - :github:`GitHub <CuriousLearner/python-docs-hi-in>`
    * - Hungarian (hu)
-     - Tamás Bajusz (:github-user:`gbtami`)
```

Comment:
cc @gbtami -- do you no longer wish to be listed as the Hungarian translation coördinator?

Reply:
Exactly.

Reply:
Thank you for your service thus far @gbtami!

A